go/api/go_api/auth.py

```diff
@@ -47,11 +52,76 @@ def requestAvatarId(self, credentials):
         raise error.UnauthorizedLogin()
 
 
+class GoAuthBouncerCredentialFactory(BasicCredentialFactory):
+    implements(ICredentialFactory)
```

Comment:
I'm trying to be consistent with the other uses in the same file. We can change all of them, but I'd rather do that separately if that's what we want to do.